eng/common/TestResources/New-TestResources.ps1

```diff
@@ -432,7 +435,20 @@ try {
     }
 
     # Deploy the templates
-    foreach ($templateFile in $templateFiles) {
+    foreach ($templateFileObject in $templateFiles) {
+        if ($templateFileObject.Extension -eq '.bicep') {
+            $templateFile = $templateFileObject.DirectoryName + '/test-resources.compiled.json'
```

Comment:
Do we want to put this file in the source tree?

Reply:
Are you asking if we should expect it to be checked in, or if we should keep the compiled output in somewhere like `/tmp`?

Reply:
I think we should keep generated files outside of the source repo so maybe tmp or somewhere like that.

Reply:
Sounds good. Updated. I did a little path construction to make the filename reflect its origin, and to handle multiple found files.